ihe-interop/src/main/java/org/motechproject/ihe/interop/domain/HL7Recipient.java

```diff
@@ -9,12 +9,22 @@
 
     private String recipientUrl;
 
+    private String recipientUsername;
+
+    private String recipientPassword;
+
     public HL7Recipient() {
     }
 
-    public HL7Recipient(String recipientName, String recipientUrl) {
+    public HL7Recipient(String recipientName, String recipientUrl, String recipientUsername, String recipientPassword) {
         this.recipientName = recipientName;
         this.recipientUrl = recipientUrl;
+        this.recipientUsername = recipientUsername;
+        this.recipientPassword = recipientPassword;
+    }
+
+    public HL7Recipient(String recipientName, String recipientUrl) {
```

Comment:
The constructors are usually ordered by the growing number of arguments they take